openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -79,7 +79,7 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                 class="subsection-text outline-button"
                                 id="${ subsection['id'] }"
                             >
-                            % if num_graded_problems:
+                            % if num_graded_problems and not 'special_exam_info' in subsection:
```

Comment:
Arguably, this should also be checking `subsection.get('format') or 'special_exam_info' in subsection`, because those are also part of the conditional the gets the icon to display lower down. Or, even better, those variables could be extracted out to outside the formatting block, so that they can be referenced in both places.